### PR TITLE
Good to go!

### PR DESCRIPTION
**Pull request by cbebout**
##Cool, right?

### CODE REVIEW

**Comment by cbebout** [comment]
hehe ... don't really get all the details here, yet;  but that's ok.  Don't need to know it all!

**Review by PWvpham — COMMENTED** [review]
CBebout-sensei, you are always approved

File: _slides/11-cbebout.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Cbebout is finally making Some progress to understand Git ;-]]
```
Comment (ArthurPricePW) [inline _slides/11-cbebout.md:6]: Now you are a grand master at Git

**Review by ArthurPricePW — APPROVED** [review]
I will ask you all of the Git Questions now

File: _slides/11-cbebout.md (hunk 1 above)
Comment (PWvpham) [reply _slides/11-cbebout.md:6]: ❤️

**Pull request merged**